src/full/Agda/TypeChecking/Errors.hs

```diff
@@ -35,6 +35,11 @@ import Data.List (sortBy, dropWhileEnd, intercalate)
 import Data.Maybe
 import qualified Data.Set as Set
 import qualified Text.PrettyPrint.Boxes as Boxes
+import Data.IntSet (IntSet)
+import qualified Data.IntSet as IntSet
+import Data.Set (Set)
+import qualified Data.Set as Set
```

Comment:
This line appears above already (duplicate).

Maybe it is a good idea to order the `Data` imports alphabetically and put them all together, so one can notice any duplications.